ui/app/AppLayouts/Profile/controls/InformationTile.qml

```diff
@@ -0,0 +1,53 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Rectangle {
+    id: root
+
+    property alias primaryText: primaryText.text
+    property alias secondaryText: secondaryText.text
+    property alias tagsModel: tags.model
+    property alias tagsDelegate: tags.delegate
+
+    height: 52
+    width: layout.width + Style.current.xlPadding
+    radius: Style.current.radius
+    border.width: 1
+    border.color: Theme.palette.baseColor2
+    color: Style.current.transparent
+
+    ColumnLayout {
+        id: layout
+        spacing: -1
+        anchors.centerIn: parent
+        StatusBaseText {
+            id: primaryText
+            font.pixelSize: 13
+            color: Theme.palette.directColor5
+            visible: text
+        }
+        StatusBaseText {
+            id: secondaryText
+            font.pixelSize: 15
+            color: Theme.palette.directColor1
+            visible: text
```

Comment:
You should set width (fillWidth) for those texts, and enable `elide` for the case when the text is too long